src/components/patch/PatchContainer.test.tsx

```diff
@@ -97,58 +96,42 @@ describe("PatchContainer", () => {
     expect(variant.state("sortedStatus")).toEqual([{ "count": 1, "status": "failed", "tasks": ["test-rest-client"] }, { "count": 5, "status": "success", "tasks": ["test-agent", "test-command", "test-rest-model", "test-rest-route", "test-auth"] }]);
   })
 
-  it("check that search returns correct results", () => {
-    const event = { currentTarget: { value: "change" } };
-    const expectedResults = ["5d700d89b237361b480b8488", "5d6ffa6897b1d37d85dae0bd", "5d6ff85e97b1d37d85dae045", "5d66d4aa97b1d3794298a82f"];
-    const notInResults = ["5d7148f497b1d36cde6c995e", "5d7018929dbe325975f6a049", "5d700fcb97b1d30d193967d0", 
-    "5d6fe06e9dbe3237b4cb7c15", "5d66f2e89dbe32552f1c91f5", "5d66f0fd9dbe32552f1c9166"];
-    const input = wrapper.find(".search-container").find(InputBase);
-    expect(input).toHaveLength(1);
-    input.prop("onChange")(event as React.ChangeEvent<HTMLInputElement>);
-    const visibleIds: string[] = [];
-    (wrapper.state("visiblePatches") as PatchInfo[]).map(patch => (
-      visibleIds.push(patch.id)
-    ));
-    for (const versionId of expectedResults) {
-      expect(visibleIds).toContain(versionId);
-    }
-    for (const versionId of notInResults) {
-      expect(visibleIds).not.toContain(versionId);
-    }
-    wrapper.setState({searchText: ""});
-  })
+  describe('Filtering pathces', () => {
+    let otherWrapper: enzyme.ReactWrapper
 
-  const checkFilteredState = jest.fn(() => {
-    expect(wrapper.state("selectedProjects")).toEqual([]);
-    expect(wrapper.state("selectedStatuses")).toEqual(["created"]);
-    const visibleIds: string[] = [];
-    const notInResults = ["5d7148f497b1d36cde6c995e", "5d700fcb97b1d30d193967d0", "5d700d89b237361b480b8488", 
-    "5d6ffa6897b1d37d85dae0bd", "5d6ff85e97b1d37d85dae045", "5d6fe06e9dbe3237b4cb7c15",
-    "5d66f2e89dbe32552f1c91f5", "5d66f0fd9dbe32552f1c9166", "5d66d4aa97b1d3794298a82f"];
-    const expectedResults = ["5d7018929dbe325975f6a049"];
-    (wrapper.state("visiblePatches") as PatchInfo[]).map(patch => {
-      visibleIds.push(patch.id);
-    });
-    for (const versionId of expectedResults) {
-      expect(visibleIds).toContain(versionId);
-    }
-    for (const versionId of notInResults) {
-      expect(visibleIds).not.toContain(versionId);
-    }
-  })
+    beforeEach(() => {
+      otherWrapper = enzyme.mount(<PatchContainer client={rest.EvergreenClient("", "", "", "", true)} username={"admin"} onFinishStateUpdate={null} />);
+    })
+    afterEach(() => {
+      otherWrapper.unmount()
```

Comment:
nit: we should be consistent about whether we're using semicolons or not. I don't have a strong opinion either way, but the existing code does have semicolons at the end of lines

Reply:
i'm going to use Prettier as an code formatter, which will add the semicolons to the file automatically. however it will also format the entire document, which will cause this PR to look like it has way more changes than it does. if it's alright with you, i'll make a subsequent PR right after this with all the Prettier reformatting that will add semicolons everywhere.

Reply:
Sounds good to me and I'll install it as well. You can push a separate commit that's just the prettyfication so that the diff is still easy to read